pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -429,7 +429,7 @@ type ServiceInstanceSpec struct {
 	// instance. This field is set by the API server and not settable by the
 	// end-user. User-provided values for this field are not saved.
 	// +optional
-	UserInfo *UserInfo `json:"userInfo"`
+	UserInfo *UserInfo `json:"userInfo,omitempty"`
```

Comment:
doesn't this cause generated file changes?

Reply:
I guess I'm about to find out!  Definitely openapi changes